Gemfile

```diff
@@ -9,7 +9,7 @@ git_source(:github) { |name| "https://github.com/#{name}.git" }
 # nicely with RVM and we should be explicit since Ruby is such a fundamental
 # part of a Rails project. The Ruby version is also locked in place by the
 # Docker base image so it won't be updated with fuzzy matching.
-ruby '2.5.8'
+ruby '3.0.2'
```

Comment:
```suggestion
ruby '3.0'
```

Reply:
Still unresolved

Reply:
Fixed

Reply:
I completely removed this line from the Gemfile.
We're building the docker image from base-image that has ruby installed (conjur-base-image).
Therefore the default ruby version will be what is defined there.
This gives us flexibility in the future as required.